doc/build.txt

```diff
@@ -26,16 +23,14 @@ the underlying build system (or its absence).
 
 ===============================================================================
 2. Commands                                                      *build-commands*
+                                                                          *Build*
 
 This plugin propose few commands:
 - `:Build` a general purpose command used to achieve common tasks
 - `:BuildInit` initialize the build system if needed
 - `:BuildInfo` provide informations and help message on the current build
   system
```

Comment:
You can remove this entire paragraph. `BuildInit` and `BuildInfo` are deprecated.